src/status_im/events.cljs

```diff
@@ -29,7 +29,7 @@
     status-im.contexts.shell.qr-reader.events
     status-im.contexts.shell.share.events
     status-im.contexts.syncing.events
-    status-im.contexts.wallet.common.wizard
+    status-im.contexts.wallet.common.wizard.events
```

Comment:
@status-im/wallet-mobile-devs 
let's keep events in events.cljs files for consistency 👍